grails-app/views/layouts/main.gsp

```diff
@@ -100,6 +100,17 @@
 
     </style>
 
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-54091417-4', 'auto');
```

Comment:
We parameterised the tracking ID in the Portal so that the production one was only injected into the Prod Portal. This means there were no events recorded for development, systest, rc, etc.
Might be an idea here?